app/src/main/java/com/miraclehwan/detektintegrationexample/FailTest.kt

```diff
@@ -0,0 +1,9 @@
+package com.miraclehwan.detektintegrationexample
+
+data class FailTest (val a:Int,val  b:Boolean,
+
+                     val c:Int){
+    fun testtest( a:Int) {}
+
+    fun maxLineWarning_1234567890_1234567890_1234567890_1234567890_1234567890_1234567890_1234567890_1234567890_1234567890( a:Int) = Unit
+}
```

Comment:
⚠️ **[detekt]** <detekt.NewLineAtEndOfFile> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The file /home/runner/work/ExampleDetekt/ExampleDetekt/app/src/main/java/com/miraclehwan/detektintegrationexample/FailTest.kt is not ending with a new line.